tst/EnergyPlus/unit/DaylightingManager.unit.cc

```diff
@@ -2305,3 +2308,560 @@ TEST_F(EnergyPlusFixture, DaylightingManager_OutputFormats)
+        "    0.08,                    !- Back Side Solar Reflectance at Normal Incidence",
+        "    0.90,                    !- Visible Transmittance at Normal Incidence",
+        "    0.08,                    !- Front Side Visible Reflectance at Normal Incidence",
+        "    0.08,                    !- Back Side Visible Reflectance at Normal Incidence",
+        "    0.00,                    !- Infrared Transmittance at Normal Incidence",
+        "    0.90,                    !- Front Side Infrared Hemispherical Emissivity",
+        "    0.90,                    !- Back Side Infrared Hemispherical Emissivity",
+        "    0.90;                    !- Conductivity {W/m-K}",
+
+        "  Material,",
+        "    Very High Reflectivity Surface,  !- Name",
+        "    Smooth,                  !- Roughness",
+        "    0.0005,                  !- Thickness {m}",
+        "    237,                     !- Conductivity {W/m-K}",
+        "    2702,                    !- Density {kg/m3}",
+        "    903,                     !- Specific Heat {J/kg-K}",
+        "    0.90,                    !- Thermal Absorptance",
+        "    0.05,                    !- Solar Absorptance",
+        "    0.05;                    !- Visible Absorptance",
+
+        "  Construction,",
+        "    EXTWALL80,               !- Name",
+        "    A1 - 1 IN STUCCO,        !- Outside Layer",
+        "    C4 - 4 IN COMMON BRICK,  !- Layer 2",
+        "    E1 - 3 / 4 IN PLASTER OR GYP BOARD;  !- Layer 3",
+
+        "  Construction,",
+        "    FLOOR SLAB 8 IN,         !- Name",
+        "    C10 - 8 IN HW CONCRETE;  !- Outside Layer",
+
+        "  Construction,",
+        "    ROOF,                    !- Name",
+        "    ROOFING - ASPHALT SHINGLES,  !- Outside Layer",
+        "    E3 - 3 / 8 IN FELT AND MEMBRANE,  !- Layer 2",
+        "    BB46 - 5 / 8 IN PLYWOOD; !- Layer 3",
+
+        "  Construction,",
+        "    CEILING IN ZONE,         !- Name",
+        "    INS - GLASS FIBER BONDED 3 IN,  !- Outside Layer",
+        "    E1 - 3 / 4 IN PLASTER OR GYP BOARD;  !- Layer 2",
+
+        "  Construction,",
+        "    CEILING IN ATTIC,        !- Name",
+        "    E1 - 3 / 4 IN PLASTER OR GYP BOARD,  !- Outside Layer",
+        "    INS - GLASS FIBER BONDED 3 IN;  !- Layer 2",
+
+        "  Construction,",
+        "    TDD Pipe,                !- Name",
+        "    Very High Reflectivity Surface;  !- Outside Layer",
+
+        "  Construction,",
+        "    TDD Dome,                !- Name",
+        "    Clear Acrylic Plastic;   !- Outside Layer",
+
+        "  Construction,",
+        "    TDD Diffuser,            !- Name",
+        "    Diffusing Acrylic Plastic;  !- Outside Layer",
+
+    });
+
+    ASSERT_TRUE(process_idf(idf_objects));
+    bool foundErrors = false;
+
+    HeatBalanceManager::GetProjectControlData(state.outputFiles, foundErrors); // read project control data
```

Comment:
Once you merge in develop, this will need a new argument: ` HeatBalanceManager::GetProjectControlData(state, state.outputFiles, ErrorsFound);`
And there are other things that won't build in this unit test now.

Reply:
I fixed all of the errors in the unit test that popped up due to all the updates since the last push.  Unit tests should all be working now.